.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,43 @@
+name: Code Coverage
+
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+    branches: [ 'master' ]
+  
+jobs:
+  coverage:
+      name: Code Coverage
+ 
+      runs-on: ubuntu-latest
+      
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+          with:
+            fetch-depth: 0
+          
+        - name: Setup PHP with Xdebug
+          uses: shivammathur/setup-php@v2
+          with:
+            php-version: '8.1'
```

Comment:
Let's use strategy matrix and create multiple jobs for different PHP versions instead of just PHP 8.1. You can check the Omise PHP Github workflow for reference.

Reference:
- https://github.com/omise/omise-php/blob/master/.github/workflows/coding-standard-check.yml
- https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
@aashishgurung we have another pipeline for unit test, may be we can modify that pipeline for different php version. But for code coverage one version is enough I think.

Reply:
Ah I thought we are modifying the same file. So, we are running test from two jobs. Maybe we can upload the XML file to the artifacts from the other job and download that XML from the artifact in this job and upload it to SoncarCloud. This way we run tests from only one job.

Let's work on that creating a different ticket. Thanks for the good work :+1: